website/src/components/code-insights/mock-data.tsx

```diff
@@ -0,0 +1,114 @@
+import { DATA_SERIES_COLORS } from './constants'
+import { CaptureGroupInsightData, SearchInsightData } from './types'
+import React from 'react'
+
+import styles from './CodeInsightsExamples.module.scss'
+
+export const SEARCH_INSIGHT_CSS_MODULES_EXAMPLES_DATA: SearchInsightData = {
+    title: 'Migration to CSS modules',
+    repositories: (
+        <>
+            <span className={styles.keyword}>repo:</span>github.com/awesomeOrg/examplerepo
+        </>
+    ),
+    data: [
+        { x: 1588965700286 - 4 * 24 * 60 * 60 * 1000, a: 88, b: 410 },
+        { x: 1588965700286 - 3 * 24 * 60 * 60 * 1000, a: 95, b: 410 },
+        { x: 1588965700286 - 2 * 24 * 60 * 60 * 1000, a: 110, b: 315 },
+        { x: 1588965700286 - 1.5 * 24 * 60 * 60 * 1000, a: 160, b: 180 },
+        { x: 1588965700286 - 1.3 * 24 * 60 * 60 * 1000, a: 310, b: 90 },
+        { x: 1588965700286 - 1 * 24 * 60 * 60 * 1000, a: 520, b: 45 },
+        { x: 1588965700286, a: 700, b: 10 },
+    ],
+    series: [
+        {
+            dataKey: 'a',
```

Comment:
Well, this warning was that one of the charts has non-unique data keys for two data series. I fixed this problem so it shouldn't be an issue anymore. But agree that data keys there could be more meaningful rather than just a/b/c ...